shell/platform/embedder/embedder_external_view_dl_op_spy.h

```diff
@@ -0,0 +1,136 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_EMBEDDER_EMBEDDER_EXTERNAL_VIEW_DL_OP_SPY_H_
+#define FLUTTER_SHELL_PLATFORM_EMBEDDER_EMBEDDER_EXTERNAL_VIEW_DL_OP_SPY_H_
+
+#include "flutter/display_list/dl_op_receiver.h"
+
+namespace flutter {
+
+//------------------------------------------------------------------------------
+/// Receives to drawing commands of a DisplayListBuilder.
+///
+/// This is used to determine whether anything will be drawn
+/// on the canvas.
+class EmbedderExternalViewDlOpReceiver final : public virtual DlOpReceiver {
```

Comment:
Have this inherit from the various "Ignore" helpers in dl_receiver_utils then you don't have to create all of the empty overrides.

You can even do that for the attributes and then override the ones you want to handle and leave the rest to be ignored by the helper.